src/amo/components/SuggestedPages/index.js

```diff
@@ -1,19 +1,41 @@
+/* @flow */
+import config from 'config';
 import * as React from 'react';
-import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import { compose } from 'redux';
 
 import Link from 'amo/components/Link';
-import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_THEME,
+  CLIENT_APP_ANDROID,
+} from 'core/constants';
 import translate from 'core/i18n/translate';
 import { visibleAddonType } from 'core/utils';
+import type { AppState } from 'amo/store';
+import type { I18nType } from 'core/types/i18n';
 
-export class SuggestedPagesBase extends React.Component {
-  static propTypes = {
-    i18n: PropTypes.object.isRequired,
+type Props = {||};
+
+type InternalProps = {|
+  ...Props,
+  _config: typeof config,
+  i18n: I18nType,
+  clientApp: string,
+|};
+
+export class SuggestedPagesBase extends React.Component<InternalProps> {
+  static defaultProps = {
+    _config: config,
   };
 
   render() {
-    const { i18n } = this.props;
+    const { _config, clientApp, i18n } = this.props;
+
+    const showThemes =
+      clientApp === CLIENT_APP_ANDROID
+        ? _config.get('enableFeatureStaticThemesForAndroid')
+        : true;
```

Comment:
is it worth it to make this a little helper function since you use this many times?

Reply:
Yep, thanks for the idea!